docs/stories/from-sphinx-to-mkdocs.md

```diff
@@ -104,6 +104,29 @@ and thrive. One notable mention is that "blog" feature seems to be for insiders
 only, for now. The others seem reasonable to me, they were cool features but not
 really what I would call essentials.
 
+### Rendering ANSI output from external commands
+
+You need the following steps
+
+- Install `pygments-ansi-color` and `markdown-exec` packages
+- Add `markdown-exec` to `plugins` in `mkdocs.yml`
+- Add `extra_css` for stylesheet like
+  [this](https://github.com/42picky/42picky.github.io/blob/main/docs/stylesheets/extra.css)
+  for the colors.
+- Use `ansi` language for code blocks or `return="ansi"` for markdown-exec.
```

Comment:
I think it's `result` instead of `return`.

Reply:
I will check to be sure. TBH, I find both confusing as I would have expected either `lang` or `formatter` as a better name.

In fact I would have wished the "ansi" was not even needed and just implied as I see zero use for displaying an ANSI stream as plain text. Still that bit is likely not in your extension control.

I will include the CSS in a plugin once I learn how to do it, to make it much easier to reuse.